grafana-plugin/src/pages/users/Users.tsx

```diff
@@ -56,9 +57,10 @@ class Users extends React.Component<UsersProps, UsersState> {
     },
 
     errorData: initErrorDataState(),
+    initialUsersLoaded: false,
   };
 
-  initialUsersLoaded = false;
+  REQUIRED_PERMISSION_TO_VIEW_USERS = UserActions.UserSettingsWrite;
```

Comment:
Shouldn't this be `UserActions.UserSettingsRead` instead? Why does it need write permissions for viewing? :thinking: 

Also, the UserActions are pretty trivial and very readable, do we really need this const here? Such constant should not belong to the class anyway, either outside of it or in the constants file, but I don't think we need it - or do we?

Reply:
on the backend, `GET /users` (list all users) currently requires `grafana-oncall-app:user-settings:write` ([or Editor basic role when RBAC is disabled](https://github.com/grafana/oncall/blob/dev/engine/apps/api/permissions/__init__.py#L175-L177))

I totally agree that the naming here is confusing, we should come back and address this specific point in a separate PR imo.

Reply:
👍 addressed [here](https://github.com/grafana/oncall/pull/1185/commits/29c0b21803c27b8557d9f0ea9d24a62b7717276b)